sorter.py

```diff
@@ -0,0 +1,73 @@
+from order import Order
+
+
+class Sorter:
```

Comment:
As far as I know sorting solutions require O(NlogN) minimally, do you think that it has this complexity?